gradle.properties

```diff
@@ -2,10 +2,14 @@ xhAppCode=toolbox
 xhAppName=Toolbox
 xhAppVersion=3.0-SNAPSHOT
 
-hoistCoreVersion=12.0-SNAPSHOT
+hoistCoreVersion=13.0-SNAPSHOT
 runHoistInline=false
 
-grailsVersion=3.3.9
-grailsAsyncVersion=3.3.2
-gormVersion=6.1.11.RELEASE
-gradleWrapperVersion=4.10.3
+grailsVersion=5.0.3
+groovyVersion=3.0.7
+gormVersion=7.1.0
+grailsGradlePluginVersion=5.0.0
+
+org.gradle.daemon=true
+org.gradle.parallel=true
+org.gradle.jvmargs=-Dfile.encoding=UTF-8 -Xmx1024M
```

Comment:
Just checking that we are sure this value makes it through OK without any extra quoting (the inner = seems like it could cause problems reading it here on GH)

Reply:
will look -- great catch -- this was copied from output from grails create-app, but good to check